app/views/admin/users/index.html.erb

```diff
@@ -29,7 +29,7 @@
             <td><%= user.likes.count %></td>
             <td><%= user.private? ? t('.private') : t('.public') %></td>
             <td><%= user.deleted_at? ? t('.deleted') : t('.active') %></td>
-            <td><%= user.provider? ? user.provider : t('.native') %></td>
+            <td><%= user.provider || t('.native') %></td>
```

Comment:
you can move it to user method

Reply:
But I can't use method "translate" in model

Reply:
```
def human_provider
  provider || I18n.t('.native')
end
```